scripts/download-overrides.py

```diff
@@ -20,34 +20,46 @@ def get_NVRA_from_NEVRA(string: str) -> str:
 
     # return the first hawkey.NEVRA item in the list of possibilities
     rpminfo = nevras[0]
-
-    # come up with rpm NVRA
-    nvra = f"{rpminfo.name}-{rpminfo.version}-{rpminfo.release}.{rpminfo.arch}"
-    return nvra
-
+    return rpminfo
 
 def is_override_lockfile(filename: str) -> bool:
     return (filename.startswith('manifest-lock.overrides.')
             and filename[-4:] in ['json', 'yaml'])
 
+def assert_epochs_match(overrides_epoch: int, rpmfile_epoch: str):
+    # normalize the input into a string
+    if overrides_epoch is None:
+       normalized_overrides_epoch = '(none)'
```

Comment:
Worth mentioning here that this matches the output of `%{E}`.

Reply:
+1

Reply:
ok I added a comment